src/components/standalone-link/standalone-link.module.css

```diff
@@ -32,6 +29,14 @@
 		margin: -5px;
 		padding: 5px;
 	}
+
+	/*
+  When rendered inside of `StandaloneLink`, prevent `StandaloneLinkContents`
+  from setting its own color.
+  */
+	& .contents {
+		color: inherit !important;
```

Comment:
🚧 If possible, I think it'd be preferable to avoid the `!important` here... easier said than done though 😅 

One way to do this might be pass in an optional "inheritColor" boolean prop to `StandaloneLinkContents`. When that prop is set to `true`, the `color` classes would not be applied.

Reply:
I like this idea! I can't remember if I already tried it last night or not; maybe it was just something similar I tried that TypeScript yelled at me about 😅